<ui version="4.0" >
 <author>fullmetalcoder</author>
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>461</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>About Edyuk</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="tabPosition" >
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>505</width>
        <height>402</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>About Edyuk</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>900</width>
           <height>600</height>
          </size>
         </property>
         <property name="sizeIncrement" >
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="pixmap" >
          <pixmap resource="../images/Edyuk.qrc" >:/splash.jpg</pixmap>
         </property>
         <property name="scaledContents" >
          <bool>true</bool>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>505</width>
        <height>402</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>License</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QTextEdit" name="eLicense" >
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="tabStopWidth" >
          <number>40</number>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>505</width>
        <height>402</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Changelog</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QTextEdit" name="eChangelog" >
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="tabStopWidth" >
          <number>40</number>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>505</width>
        <height>402</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Thanks</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QTextEdit" name="eThanks" >
         <property name="lineWrapMode" >
          <enum>QTextEdit::WidgetWidth</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="tabStopWidth" >
          <number>40</number>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2" />
 <resources>
  <include location="../images/Edyuk.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AboutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>267</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel" >
     <x>237</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
